firebase-firestore/CHANGELOG.md

```diff
@@ -1,17 +1,19 @@
 # Unreleased
+
+# 18.0.0
 - [changed] The `timestampsInSnapshotsEnabled` setting is now enabled by
-  default so timestamp fields read from a `DocumentSnapshot` will be returned
-  as `Timestamp` objects instead of `Date`. Any code expecting to receive a
-  `Date` object must be updated.
-- [feature] Custom objects (POJOs) can now be passed as a field value in
-  update(), within `Map<>` objects passed to set(), in array transform
-  operations, and in query filters.
+  default. Timestamp fields taht read from a `DocumentSnapshot` are now
+  returned as `Timestamp` objects instead of `Date` objects. This is a breaking
+  change; developers must update any code that expects to receive a `Date`
+  object. See https://firebase.google.com/docs/reference/android/com/google/firebase/firestore/FirebaseFirestoreSettings.Builder.html#setTimestampsInSnapshotsEnabled(boolean) for more details.
```

Comment:
Seems like this is something we should port into the iOS release notes too?

Reply:
Yep. I'll adjust the other ports shortly.

Reply:
https://github.com/firebase/firebase-ios-sdk/pull/2286
https://github.com/firebase/firebase-js-sdk/pull/1484